extensions/wikia/Wall/WallHelper.class.php

```diff
@@ -572,7 +572,8 @@ public static function getWallTitleData( $rc = null, $row = null ) {
 
 		} else {
 			$objTitle = $rc->getTitle();
-			$userText = User::getUsername( $rc->getAttribute( 'rc_user' ), $rc->getAttribute( 'rc_user_text' ) );
+			$userIp = inet_ntop( $rc->getAttribute( 'rc_ip_bin' ) );
+			$userText = User::getUsername( $rc->getAttribute( 'rc_user' ), $userIp );
 			$isNew = false; // it doesn't metter for rc -- we've got there rc_log_action
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app